accumulo_plugin/plugin-accumulo/testfiles/PelzCryptoTest.java

```diff
@@ -306,30 +379,35 @@ public void testReadNoCryptoWithCryptoConfigured() throws Exception {
       }
     }
 
-    Scanner iter =
-        RFile.newScanner().from(file).withFileSystem(fs).withTableProperties(cryptoOnConf).build();
-    ArrayList<Key> keysRead = new ArrayList<>();
-    iter.forEach(e -> keysRead.add(e.getKey()));
+    ArrayList<Key> keysRead;
+    try (Scanner iter = RFile.newScanner().from(file).withFileSystem(fs)
+        .withTableProperties(cryptoOnConf).build()) {
+      keysRead = new ArrayList<>();
+      iter.forEach(e -> keysRead.add(e.getKey()));
+    }
     assertEquals(keys, keysRead);
   }
 
   @Test
   public void testMissingConfigProperties() throws ReflectiveOperationException {
-    ConfigurationCopy aconf = new ConfigurationCopy(DefaultConfiguration.getInstance());
-    Configuration conf = new Configuration(false);
-    for (Map.Entry<String,String> e : conf) {
-      aconf.set(e.getKey(), e.getValue());
-    }
-    aconf.set(Property.INSTANCE_CRYPTO_SERVICE,
-        "org.apache.accumulo.core.pelz.PelzCryptoService");
-    String configuredClass = aconf.get(Property.INSTANCE_CRYPTO_SERVICE.getKey());
-    Class<? extends CryptoService> clazz =
-        ClassLoaderUtil.loadClass(configuredClass, CryptoService.class);
-    CryptoService cs = clazz.getDeclaredConstructor().newInstance();
+    var cryptoProps = getAllCryptoProperties(ConfigMode.CRYPTO_TABLE_ON);
+    var droppedProperty = cryptoProps.remove(PelzCryptoService.KEY_URI_PROPERTY);
+    assertNotNull(droppedProperty);
 
-    assertEquals(PelzCryptoService.class, cs.getClass());
-    assertThrows(NullPointerException.class,
-        () -> cs.init(aconf.getAllPropertiesWithPrefix(Property.TABLE_PREFIX)));
+    String configuredClass = cryptoProps.get(INSTANCE_CRYPTO_FACTORY.getKey());
+    CryptoEnvironment env = new CryptoEnvironmentImpl(TABLE, TableId.of("5"), null);
```

Comment:
What does the magic 5 mean here?

Reply:
This is a copy over from the '''CryptoTest.java''' written by the Accumulo team.